src/app/camping-vliegenbos/over-de-camping/page.tsx

```diff
@@ -0,0 +1,295 @@
+'use client'
+
+import {
+  Accordion,
+  Alert,
+  Breadcrumb,
+  DescriptionList,
+  Grid,
+  Heading,
+  Link,
+  LinkList,
+  Paragraph,
+  Spotlight,
+  StandaloneLink,
+  Tabs,
+  UnorderedList,
+} from '@amsterdam/design-system-react'
+import NextImage from 'next/image'
+
+import formatPath from '../../../utils/formatPath'
+import pictureCampingImage from '../_assets/camping-vliegenbos-picture.jpg'
+import mapCampingImage from '../_assets/map_camping.jpg'
+
+export default function OverDeCamping() {
+  return (
+    <>
+      <Grid as="section" paddingBottom="large">
+        <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }}>
+          <Breadcrumb className="ams-mb-m">
```

Comment:
Breadcrumb renders a `nav` that should sit between `header` and `main`. That’s why we render a single Grid around it (for now). This cell, and the one for the breadcrumb, should also be a bit [wider on desktop](https://designsystem.amsterdam/?path=/docs/pages-guidelines--docs#content-page-header).